include/ceed/backend.h

```diff
@@ -32,6 +32,39 @@
 #define CEED_COMPOSITE_MAX 16
 #define CEED_EPSILON 1E-16
 
+/**
+  @ingroup Ceed
+  This macro provides the ablitiy to disable optimization flags for functions that
+  are sensitive to floting point optimizations.
+**/
+#ifndef CeedPragmaOptimizeOff
+#  if defined(__clang__)
+#    define CeedPragmaOptimizeOff _Pragma("clang optimize off")
+#  elif defined(__GNUC__)
+#    define CeedPragmaOptimizeOff _Pragma("GCC push_options") _Pragma("GCC optimize 0")
+#  elif defined(__INTEL_COMPILER)
+#    define CeedPragmaOptimizeOff _Pragma('optimize("", off)')
```

Comment:
Can single quote possibly be correct here?

Reply:
No, it can't. The intel job compiled, so perhaps that job does not believe that `__INTEL_COMPILER` is defined?

Reply:
I think its ok now